sys/arduino/include/serialport.hpp

```diff
@@ -84,7 +84,7 @@ class SerialPort {
      *
      * @param[in] speed     speed in bits per second (baud)
      */
-    void begin(int speed);
+    void begin(uint32_t speed);
```

Comment:
actually, the original arduino API specifies this to `long`, so shouldn't we use that type here (as this this the implemenation of the arduino API)?